.github/workflows/build-shop.yml

```diff
@@ -80,13 +86,13 @@ jobs:
       - name: Pull images in background
         working-directory: ${{ env.PS_DIR }}
         run: |
-          # Pull mysql image
-          USER_ID=$(id -u) GROUP_ID=$(id -g) nohup docker compose -f docker-compose.yml pull -q mysql >& /dev/null &
+          # Pull ${{ inputs.db_type }} image
+          USER_ID=$(id -u) GROUP_ID=$(id -g) nohup docker compose -f docker-compose.yml pull -q ${{ inputs.db_type }} >& /dev/null &
       - name: Build PrestaShop image in background
         working-directory: ${{ env.PS_DIR }}
         run: |
           # Build prestashop image in background
-          USER_ID=$(id -u) GROUP_ID=$(id -g) nohup docker compose -f docker-compose.yml build >& /dev/null &
+          USER_ID=$(id -u) GROUP_ID=$(id -g) nohup docker compose -f docker-compose.yml build prestashop-git >& /dev/null &
```

Comment:
I believe the dependency you setup in the docker file depends on the env variable DB_SERVICE/DB_SERVER I guess you could use the build without specifying `prestashop-git` if you set this env variable no?